src/pages/SelectServersToProvision.js

```diff
@@ -25,7 +25,7 @@ import { InputLine } from '../components/InputLine.js';
 import { PlaybookProgress } from '../components/PlaybookProcess.js';
 import { fetchJson } from '../utils/RestUtils.js';
 
-const INSTALL_PLAYBOOK = 'dayzero-os-provision';
+const INSTALL_PLAYBOOK = 'installui-os-provision';
```

Comment:
Ditto. Dang, how many times are we going to define constants for these same playbooks? It would be better to define the centrally in one place, like `utils/constants.js` and use them throughout the code base.